client/i18n/locales/espanol/translations.json

```
@@ -416,63 +416,63 @@
-    "msg-44": "Comprueba si has proporcionado un nombre de usuario y un informe",
-    "msg-45": "Se envió un informe al equipo con {{email}} en copia"
+    "honest-first": "Para reclamar una certificación, primero debes aceptar nuestra política de honestidad académica.",
+    "really-weird": "Algo realmente extraño sucedió. Si vuelve a ocurrir, considera hacer un reporte del problema en https://github.com/freeCodeCamp/freeCodeCamp/issues/new",
+    "not-right": "Algo no está bien. Se ha generado un informe y se ha notificado al equipo de freeCodeCamp.org",
+    "went-wrong": "Algo salió mal, verifica e intenta nuevamente",
+    "account-deleted": "Tu cuenta ha sido eliminada con éxito",
+    "progress-reset": "Tu progreso se ha restablecido",
+    "not-authorized": "No estás autorizado a continuar en esta ruta",
+    "could-not-find": "No pudimos encontrar lo que buscabas. Por favor revisa e intenta de nuevo",
+    "wrong-updating": "Se produjo un error al actualizar tu cuenta. Por favor revisa e intenta de nuevo",
+    "updated-preferences": "Hemos actualizado tus preferencias",
+    "email-invalid": "El formato del correo electrónico no es válido",
+    "bad-challengeId": "currentChallengeId no es un ID de desafío válido",
+    "theme-invalid": "El tema no es válido",
+    "theme-set": "Tema ya establecido",
+    "theme-updated": "¡Tu tema ha sido actualizado!",
+    "username-used": "El nombre de usuario ya está asociado a esta cuenta",
+    "username-taken": "El nombre de usuario ya está asociado a una cuenta diferente",
+    "username-updated": "Hemos actualizado tu nombre de usuario a {{username}}",
+    "could-not-logout": "No pudimos cerrar la sesión. Vuelve a intentarlo en un momento.",
+    "email-encoded-wrong": "El correo electrónico codificado en el enlace tiene un formato incorrecto",
+    "oops-not-right": "¡uy!, algo no está bien, solicite un enlace nuevo para iniciar sesión / registrarse",
+    "expired-link": "Parece que el enlace en el que hiciste clic ha expirado, solicita un enlace nuevo para iniciar sesión",
+    "signin-success": "¡Éxito! Ha iniciado sesión en su cuenta. ¡Feliz día programando!",
+    "social-auth-gone": "Nos estamos alejando de la autenticación social por razones de privacidad. La próxima vez, te recomendamos que utilices tu dirección de correo electrónico: {{email}} para iniciar sesión.",
+    "name-needed": "Necesitamos tu nombre para poder incluirlo en tu certificación. Agregue tu nombre a la configuración de tu cuenta y haz clic en el botón Guardar. Entonces podemos emitir tu certificación.",
+    "incomplete-steps": "Parece que no haz completado los pasos necesarios. Completa los proyectos requeridos para reclamar la Certificación {{name}}",
+    "already-claimed": "Parece que ya reclamó la Certificación {{name}}",
+    "cert-claim-success": "@{{usernane}}, has reclamado con éxito el {{name}} Certificación! ¡Felicitaciones en nombre del equipo de freeCodeCamp.org!",
```

```suggestion
    "cert-claim-success": "@{{username}}, has reclamado con éxito el {{name}} Certificación! ¡Felicitaciones en nombre del equipo de freeCodeCamp.org!",
```
Not caused by this PR, but since we're in here anyway...